<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Healpix Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/3dview.gif</normaloff>:/3dview.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="WorkSpace" name="workspace" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>4</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>316</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="histogramTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Histogram</string>
           </attribute>
           <widget class="HistogramWidget" name="histogramControl" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>9</y>
              <width>278</width>
              <height>884</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="optionstab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>Options</string>
           </attribute>
           <widget class="MapOptions" name="mapOptions" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>280</width>
              <height>391</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="managerWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="ViewportManager" name="viewportManager" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Map"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSynchronization">
    <property name="title">
     <string>Sync</string>
    </property>
    <addaction name="actionSync_all_maps"/>
    <addaction name="actionSync_selected_maps"/>
    <addaction name="actionStop_sync"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <widget class="QMenu" name="menuPixel_Selection">
     <property name="title">
      <string>Pixel Selection</string>
     </property>
     <addaction name="actionSingleSelection"/>
     <addaction name="actionDiscSelection"/>
     <addaction name="actionTriangleSelection"/>
     <addaction name="actionPolygonSelection"/>
    </widget>
    <addaction name="actionSelectAllViewports"/>
    <addaction name="actionClearViewportSelection"/>
    <addaction name="menuPixel_Selection"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuProjection">
     <property name="title">
      <string>Projection</string>
     </property>
     <addaction name="action3D_Sphere"/>
     <addaction name="actionMollweide"/>
    </widget>
    <addaction name="actionGrid"/>
    <addaction name="actionShow_polarization_vectors"/>
    <addaction name="actionReset_view"/>
    <addaction name="menuProjection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelect"/>
   <addaction name="menuView"/>
   <addaction name="menuSynchronization"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionReset"/>
   <addaction name="action3Dview"/>
   <addaction name="actionMollview"/>
   <addaction name="actionSynchronize"/>
   <addaction name="actionColors"/>
   <addaction name="actionPolarizationVectors"/>
   <addaction name="actionShowGrid"/>
   <addaction name="actionSelectAll"/>
   <addaction name="actionDeselectAll"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/open.gif</normaloff>:/open.gif</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Map</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/reset.gif</normaloff>:/reset.gif</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>&amp;Reset Views</string>
   </property>
  </action>
  <action name="action3Dview">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/3dview.gif</normaloff>:/3dview.gif</iconset>
   </property>
   <property name="text">
    <string>3D view</string>
   </property>
   <property name="toolTip">
    <string>Switch to 3D</string>
   </property>
  </action>
  <action name="actionMollview">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/mollview.gif</normaloff>:/mollview.gif</iconset>
   </property>
   <property name="text">
    <string>Mollview</string>
   </property>
   <property name="toolTip">
    <string>Switch to Mollview</string>
   </property>
  </action>
  <action name="actionSynchronize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/sync.gif</normaloff>:/sync.gif</iconset>
   </property>
   <property name="text">
    <string>Synchronize views</string>
   </property>
   <property name="toolTip">
    <string>Synchronize views</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/colormap.gif</normaloff>:/colormap.gif</iconset>
   </property>
   <property name="text">
    <string>Colors</string>
   </property>
   <property name="toolTip">
    <string>Show Options</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/extract.gif</normaloff>:/extract.gif</iconset>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
   <property name="toolTip">
    <string>Extract ROI</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/select-all.gif</normaloff>:/select-all.gif</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All Viewports</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/deselect-all.gif</normaloff>:/deselect-all.gif</iconset>
   </property>
   <property name="text">
    <string>Deselect All</string>
   </property>
   <property name="toolTip">
    <string>Deselect All</string>
   </property>
  </action>
  <action name="actionPolarizationVectors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pol-vectors.gif</normaloff>:/pol-vectors.gif</iconset>
   </property>
   <property name="text">
    <string>Polarization Vectors</string>
   </property>
   <property name="toolTip">
    <string>Show/Hide Polarization Vectors</string>
   </property>
  </action>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/grid.gif</normaloff>:/grid.gif</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Show Grid</string>
   </property>
  </action>
  <action name="actionOpen_Map">
   <property name="text">
    <string>Open Map</string>
   </property>
  </action>
  <action name="actionShow_polarization_vectors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show polarization vectors</string>
   </property>
  </action>
  <action name="actionSelect_all_viewports">
   <property name="text">
    <string>Select all viewports</string>
   </property>
  </action>
  <action name="actionClear_viewport_selection">
   <property name="text">
    <string>Clear viewport selection</string>
   </property>
  </action>
  <action name="actionReset_view">
   <property name="text">
    <string>Reset view</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionSync_all_maps">
   <property name="text">
    <string>Sync all maps</string>
   </property>
  </action>
  <action name="actionSync_selected_maps">
   <property name="text">
    <string>Sync selected maps</string>
   </property>
  </action>
  <action name="actionStop_sync">
   <property name="text">
    <string>Stop sync</string>
   </property>
  </action>
  <action name="actionSelectAllViewports">
   <property name="text">
    <string>All viewports</string>
   </property>
  </action>
  <action name="actionClearViewportSelection">
   <property name="text">
    <string>Deselect Viewports</string>
   </property>
  </action>
  <action name="action3D_Sphere">
   <property name="text">
    <string>3D Sphere</string>
   </property>
  </action>
  <action name="actionMollweide">
   <property name="text">
    <string>Mollweide</string>
   </property>
  </action>
  <action name="actionSingleSelection">
   <property name="text">
    <string>Single Selection</string>
   </property>
  </action>
  <action name="actionDiscSelection">
   <property name="text">
    <string>Disc</string>
   </property>
  </action>
  <action name="actionTriangleSelection">
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionPolygonSelection">
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HistogramWidget</class>
   <extends>QWidget</extends>
   <header>histogramwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewportManager</class>
   <extends>QWidget</extends>
   <header>viewportmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapOptions</class>
   <extends>QWidget</extends>
   <header>mapoptions.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorkSpace</class>
   <extends>QWidget</extends>
   <header>workspace.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
